src/biblenotify/core/notifications.py

```diff
@@ -92,11 +94,29 @@ def loadChapter(self, location: str) -> dict:
             return ["", ""]
 
         contents = QTextStream(file)
-        contents_string = contents.readAll()
+        contentsString = contents.readAll()
 
-        contents_json = json.loads(contents_string)
+        verse = self.currentVerse["verse"]
+        contentsJson = json.loads(contentsString)
+
+        chapter = contentsJson
+        chapterPlace = chapter["read"][0]["chapter"]
+
+        highlightedChapter = self.highlightVerse(verse, chapter)
 
         return {
-            "text": contents_json["read"][0]["text"],
-            "place": contents_json["read"][0]["chapter"]
+            "text": highlightedChapter,
+            "place": chapterPlace
         }
+
+    def highlightVerse(self, verse, chapter):
+        chapterText = chapter["read"][0]["text"]
+        
+        verseMatch = re.search(verse, chapterText)
```

Comment:
We should escape the search text like ``re.escape(verse)``

Reply:
Ok